doc/src/manual/environment-variables.md

```diff
@@ -190,19 +190,23 @@ a master process to establish a connection before dying.
 ### [`JULIA_NUM_THREADS`](@id JULIA_NUM_THREADS)
 
 An unsigned 64-bit integer (`uint64_t`) that sets the maximum number of threads
-available to Julia. If `$JULIA_NUM_THREADS` exceeds the number of available
-CPU threads (logical cores), then the number of threads is set to the number of CPU threads. If
-`$JULIA_NUM_THREADS` is not positive or is not set, or if the number of CPU
-threads cannot be determined through system calls, then the number of threads is
-set to `1`.
+available to Julia.  If `$JULIA_NUM_THREADS` is not positive or is not set, or
```

Comment:
The change to this paragraph should be backported to 1.6.

Reply:
Sounds good. I'll make a PR to `release-1.6` to fix just the part of the paragraph that applies to 1.6.

Reply:
Here's the PR for 1.6: https://github.com/JuliaLang/julia/pull/39118